CaptureSDK/GiniCaptureSDK/Sources/GiniCaptureSDK/Core/Screens/Camera/Camera.swift

```diff
@@ -107,47 +107,40 @@ final class Camera: NSObject, CameraProtocol {
             self.session.commitConfiguration()
         }
     }
-    
+
     func setup(completion: @escaping ((CameraError?) -> Void)) {
-        
-        setupCaptureDevice() { [weak self] result in
```

Comment:
[Clarification] Do setupCaptureDevice(){} and setupCaptureDevice{} mean the same?🧐

Reply:
Yes 😄